src/sentry/models/dashboard_widget.py

```diff
@@ -133,6 +133,8 @@ class DashboardWidgetQuery(Model):
     order = BoundedPositiveIntegerField()
     date_added = models.DateTimeField(default=timezone.now)
     date_modified = models.DateTimeField(default=timezone.now)
+    # Whether this query is hidden from the UI, used by metric widgets
+    is_hidden = models.BooleanField(null=True)
```

Comment:
Does it make sense to make this not null and default to False? https://develop.sentry.dev/database-migrations/#adding-columns-with-a-default

Reply:
It does! Thanks for pointing it out! I updated the PR, would you mind taking another quick look? thanks!